### DISCUSSION TITLE
Virus detections for Windows regular & portable version downloads?

### DISCUSSION THREAD

**Discussion by Sony-dev** [TagStudioDev/TagStudio#294] [General]
![](https://i.imgur.com/zEYCgXe.png)
Windows defender tried nuking the files after download so I ran them through VirusTotal which resulted in 11 detections for the portable version, and 13 for the regular version.

[Link to VirusTotal results for portable download](https://www.virustotal.com/gui/file/c37d2a823bc7dc675e70766ec9f1f34f9d63f436ca6aca25cfeb36f712f7cb88/detection)
[Link to VirusTotal results for regular download](https://www.virustotal.com/gui/file/36a49cdf5fc670b25c730dd19ccb6cd46388730fc2eb5da6d1f9fef514f0413c/detection)

I suspect there might be some wonky heuristics false-positives at play here, but I wanted to be sure before running anything.
Any ideas what might be causing this?

**Comment by gregor202** [comment]
Chrome as well is not accepting it and blocks the download!

**Comment by Loran425** [comment]
This is a side effect of PyInstaller, sadly as an open source tool not everyone using PyInstaller has good intentions so programs "frozen" or "packaged" with it can be flagged as false positives.
